score.py

```diff
@@ -0,0 +1,101 @@
+# this file was generated from score.ipynb
+import uuid
+import os
+import pandas as pd
+from sklearn.pipeline import make_pipeline
+import mlflow
+from sklearn.metrics import mean_squared_error
+import sys
+from prefect import flow, task
+sys.path.append(
+    "c:\\users\\user\\appdata\\local\\programs\\python\\python39\\lib\\site-packages")
+
```

Comment:
I have a concern with this, hardcoding paths is not best practice, if you're running this with a virtual environment I don't see why you need to use this. No one will be able to run your code because not everyone has their path defined in this manner